sources/assets/shells/zshrc

```diff
@@ -20,7 +20,21 @@ export GOPATH=$(go env GOPATH)
 source $ZSH/oh-my-zsh.sh
 
 TIME_="%{$fg[white]%}[%{$fg[red]%}%D{%b %d, %Y - %T (%Z)}%{$fg[white]%}]%{$reset_color%}"
-PROMPT="$LOGGING$TIME_%{$FX[bold]$FG[013]%} $EXEGOL_HOSTNAME %{$fg_bold[blue]%}%(!.%1~.%c) $(prompt_char)%{$reset_color%} "
+
+update_prompt() {
+  DB_PROMPT=""
+
+  if [[ ! -z "${USER}" ]]; then
+    DB_PROMPT="%{$fg[white]%}[$fg[yellow]%}${USER}%{$fg[white]%}]%{$reset_color%}"
+  
```

Comment:
missing a `fi`, and you need to make it so that if DOMAIN is set but not USER then nothing should be shown imo